crates/router/src/core/errors/user.rs

```diff
@@ -73,6 +73,10 @@ pub enum UserErrors {
     #[error("TotpRequired")]
     TotpRequired,
     #[error("InvalidRecoveryCode")]
+    MaxTotpAttemptsReached,
+    #[error("MaxTotpAttemptsReached")]
+    MaxRecoveryCodeAttemptsReached,
+    #[error("MaxRecoveryCodeAttemptsReached")]
```

Comment:
It's better to place these at last and follow the order of error codes when defining errors.

And also change the error attribute message to match the one defined in `get_error_message` function.

Reply:
And I think one error should be sufficient in this case, you can change this to something like this:
```rust
    MaxAttemptsReached(String), // String is the resource can be TOTP or RC.
```